package.json

```diff
@@ -23,7 +23,7 @@
     "test": "jest",
     "test:e2e": "cypress run -C cypress.config.e2e.ts",
     "test:e2e:ui": "set -a && source e2e.env && set +a && cypress open --e2e --browser electron -C cypress.config.e2e.ts --config baseUrl=https://approved-premises-dev.hmpps.service.justice.gov.uk",
-    "test:integration": "start-server-and-test start-feature http://localhost:3007/ping int-test",
+    "test:integration": "npm run start-test-wiremock && start-server-and-test start-feature http://localhost:3007/ping int-test",
```

Comment:
I nearly did this tidy up myself, then backed out given that `test:integration` is run in `script/test`, and I wasn't sure what would happen if that script spun up wiremock via `docker-compose`, spun wiremock up again via `test:integration`, then only cleaned up the latter. Happy that this is safe?

Reply:
I think it's ok? It looks to me like both `test:integration` and `script/test` attempt to start the backing services via docker-compose but only `script/test` cleans up after itself. 

If `script/test` runs after a lone request to do `test:integration` then it should find existing docker containers and continue as normal. I've given this a test and it seems to be happy!